news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,208 @@
+- [End Word](#end-word)
+
+## What's new in osu!? 
+
+Here are the latest updates, events, and more!
+
+### ![](/wiki/shared/mode/osu.png) osu! 
+
+#### The Roundtable
+
+If you ask us what is the big event of July, it'd be the [Roundtable](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)!
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_title.png)
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_players.jpg)
+
+With the players showcased above, we have [BTMC](https://osu.ppy.sh/users/3171691) and [Happystick](https://osu.ppy.sh/users/256802) commentating, a genius roster that makes sense for this first **IRL osu! invitational**!
+
+As if it wasn't enough, [mrekk](https://osu.ppy.sh/users/7562902) was there too! If you weren't able to follow the event, no worries! The official Roundtable's YouTube channel has the full VODs [here](https://www.youtube.com/channel/UC6fSQd4oKBPxkBR93GucCUg/featured)!
+
+With that many great players, such as [mrekk](https://osu.ppy.sh/users/7562902), [Utami](https://osu.ppy.sh/users/7512553), [RyuK](https://osu.ppy.sh/users/6304246), and many others, this tournament was incredible! The competition's spirit was mixed with the entertaining cast and gimmicks of [BTMC](https://osu.ppy.sh/users/3171691) and the result was nothing but epic!
+
+In terms of epic, we were served with the finals: [mrekk](https://osu.ppy.sh/users/7562902) vs [Utami](https://osu.ppy.sh/users/7512553)! Despite mrekk's defeat against [Utami]([Utami](https://osu.ppy.sh/users/7512553)), his performances were impressive, along with [Utami](https://osu.ppy.sh/users/7512553)'s. His first rank on the leaderboard is indeed his throne.
+
+A best-of from this match was made on the [BTMC](https://osu.ppy.sh/users/3171691) [channel](https://twitch.tv/btmc), with exclusive interviews of the players, by [BTMC](https://osu.ppy.sh/users/3171691) (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!
+
+Anyway, The Roundtable has now ended, we can't wait until the next edition!
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The Roundtable will return.<br>Thank you, everyone ❤️</p>&mdash; BTMC (@btmclive) <a href="https://twitter.com/btmclive/status/1546335465518682112?ref_src=twsrc%5Etfw">July 11, 2022</a></blockquote>
+
+[*Find the news article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)
+
+Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+
+#### EGTS 2022
+
+As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of [*The Roundtable*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) with their very own annual entry of **[Expert Global Taiko Showdown 2022](/wiki/en/Tournaments/GTS/EGTS_2022)**!
```

Comment:
```suggestion
As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, members of the osu!taiko community take it upon themselves to rival even that of [*The Roundtable*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) with their very own annual entry of **[Expert Global Taiko Showdown 2022](/wiki/en/Tournaments/GTS/EGTS_2022)**!
```
Reads a bit better to me, as these events are made possible by the community and not through the game mode. Using the pronouns consistently was my main concern here (see `itself`, `themselves` and `their`).